linodecli/plugins/obj/objects.py

```diff
@@ -91,7 +91,7 @@ def upload_object(
     bucket = parsed.bucket
     if "/" in parsed.bucket:
         bucket = parsed.bucket.split("/")[0]
-        prefix = parsed.bucket.lstrip(f"{bucket}/")
+        prefix = parsed.bucket[len(f"{bucket}/"):]
```

Comment:
```suggestion
        prefix = parsed.bucket.removeprefix(f"{bucket}/")
```
Thanks for your contribution! I think `removeprefix` would do a better job.

Reply:
Sure thing, not fully familiar with all the internal python string functions. I'll commit the changes to use `removeprefix`